src/annotationLayer.js

```diff
@@ -6,25 +6,41 @@ var registry = require('./registry');
 var transform = require('./transform');
 var $ = require('jquery');
 var Mousetrap = require('mousetrap');
+var textFeature = require('./textFeature');
+
+/**
+ * @typedef {object} geo.annotationLayer.labelRecord
+ * @property {string} text The text of the label
+ * @property {geo.geoPosition} position The position of the label in map gcs
+ *      coordinates.
+ * @property {object} [style] A `geo.textFeature` style object.
+ */
 
 /**
  * Layer to handle direct interactions with different features.  Annotations
  * (features) can be created by calling mode(<name of feature>) or cancelled
  * with mode(null).
  *
- * @class geo.annotationLayer
+ * @class
+ * @alias geo.annotationLayer
  * @extends geo.featureLayer
- * @param {object?} options
- * @param {number} [options.dblClickTime=300]  The delay in milliseconds that
- *    is treated as a double-click when working with annotations.
- * @param {number} [options.adjacentPointProximity=5]  The minimum distance in
+ * @param {object} [args] Layer options.
+ * @param {number} [args.dblClickTime=300] The delay in milliseconds that is
```

Comment:
I am wondering if 300ms is longer than usual but this is not something for this PR